docs/README.md

```diff
@@ -29,40 +29,40 @@ For support and questions, visit [the NeoForged Discord server][Discord].
 
```

Comment:
Should probably rewrite this original installing NeoForge thing as it doesn't make sense and should say how to install NeoForge without going through Curseforge and linking to the installer on https://neoforged.net/